README.md

```diff
@@ -165,14 +168,78 @@ Network mode is set to `host` to allow the prover node to connect to the mongodb
 
 If you are unsure about modifying or customizing changes, refer to the section below.
 
+### Initializing from Checkpoint
+
+Using our custom `mongo` image, we need to initialize the database and restore from a checkpoint.
+
+If you run the mongodb service with default options, there is no need to configure anything as the checkpointed database will be initialized and restored automatically.
+
+#### Network Mode: Host
```

Comment:
Mostly likely our public prover holder will use default so we do not make them upset.
This part may be just for if they are not using default ports so need change the title to like "For non-default/custom config" what we need to do